views/js/qtiCreator/plugins/panel/outcomeEditor.js

```diff
@@ -121,26 +184,73 @@ define([
                     $identifierInput.val('');
                     $identifierInput.val(outcome.id());
 
+                    const $outcomeValueContainer = $outcomeContainer.find('div.minimum-maximum');
+
+                    const showScoringTraitWarningOnInvalidValue = () => {
+                        if(!isValidScoringTrait(outcome.attr('normalMinimum')) || !isValidScoringTrait(outcome.attr('normalMaximum'))) {
+                            $outcomeValueContainer.data('$tooltip').show();
+                        } else {
+                            $outcomeValueContainer.data('$tooltip').hide();
+                        }
+                    };
+
+                    //Attach scoring trait warning tooltips on init to outcome value fields on init
+                    if(isScoringTraitValidationEnabled) {
+                        attachScoringTraitWarningTooltip($outcomeValueContainer);
+
+                        // shows tooltips in case of invalid value
+                        showScoringTraitWarningOnInvalidValue();
+                    }
+
                     //attach form change callbacks
                     formElement.setChangeCallbacks($outcomeContainer, outcome, _.assign({
-                        identifier : function(outcome, value){
+                        identifier : (outcome, value) => {
                             //update the html for real time update
                             $identifierLabel.html(value);
 
                             //save to model
                             outcome.id(value);
                         },
-                        interpretation : function(outcome, value){
+                        interpretation : (outcome, value) => {
```

Comment:
Same here. We should avoid to use arrow function as object's method. Please use object shorthand instead